pygmt/tests/test_helpers.py

```diff
@@ -32,25 +25,6 @@ def test_load_static_earth_relief():
     assert isinstance(data, xr.DataArray)
 
 
-@pytest.mark.parametrize(
```

Comment:
This test is removed because:

1. `data_kind` no longer checks if the input is valid
2. All these tests are already covered by the `validate_data_input` doctests.